website/docs/docs/trusted-adapters.md

```diff
@@ -4,7 +4,7 @@ id: "trusted-adapters"
 hide_table_of_contents: true
 ---
 
-Trusted adapters are adapters not maintained by dbt Labs, that we feel comfortable recommending to users for use in production.
+Trusted adapters are adapters that we feel comfortable recommending to users for use in production. They are maintained by dbt Labs, partners, and community members. For all adapters in dbt Cloud, they have gone through an additional rigorous process that cover development, documentation, user experience, and maintenance requirements.
```

Comment:
Same here as above. The language could be viewed as a slight towards community adapters. Maybe something like this:

```suggestion
Trusted adapters have undergone a rigorous vetting process to ensure a level of function and stability that makes them easy to recommend in any production environment. They are maintained by dbt Labs, partners, and community members. All adapters in dbt Cloud have gone through the trusted adapter process that covers development, documentation, user experience, and maintenance requirements.
```

Reply:
See above - trusted adapters are opt-in but trusted adapters in dbt Cloud are actually vetted

Reply:
per feedback and convo, drafted something for this paragraph. lemme know what you think!